ui/components/ui/qr-code-view/index.scss

```
@@ -6,29 +6,71 @@
   justify-content: center;
   align-items: center;
 
+  &__wrapper {
+    position: relative;
+    display: inline-block;
+  }
+
+  &__image {
+    position: relative;
+    z-index: 1;
+    border: 1px solid var(--color-border-muted);
+    border-radius: 16px;
+    background-color: white;
```

Just checking we want the background color to remain white regardless of theme here? Can we use a design token CSS var?

```suggestion
    background-color: var(--brand-colors-white); // stay white regardless of theme
```